nipype/utils/config.py

```diff
@@ -94,6 +97,15 @@ def __init__(self, *args, **kwargs):
         if os.path.exists(config_dir):
             self._config.read([config_file, 'nipype.cfg'])
 
+        for option in CONFIG_DEPRECATIONS:
+            for section in ['execution', 'logging']:
+                if self._config.has_option(section, option):
+                    new_option = CONFIG_DEPRECATIONS[option][0]
+                    if not self._config.has_option(section, new_option):
+                        # Warn implicit in get
+                        self._config.set(section, new_option,
+                                         self._config.get(option))
```

Comment:
If you want this implicit in `get`, what you probably want the local `get`:

```Python
self.set(section, new_option, self.get(section, option))
```

Also, you're missing the section in this `get`, in either case.